pydata_sphinx_theme/static/css/theme.css

```diff
@@ -0,0 +1,76 @@
+/*****************************************************************************
+ * Custom font
+**/
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+:root {
+  /*****************************************************************************
+  * Font size
+  **/
+  --font-size-base: 15px; /* base font size - applied at body / html level */
+
+  /* heading font sizes */
+  --font-size-h1: 36px;
+  --font-size-h2: 32px;
+  --font-size-h3: 26px;
+  --font-size-h4: 21px;
+  --font-size-h5: 18px;
+  --font-size-h6: 16px;
+
+  /* smaller then heading font sizes*/
+  --font-size-milli: 12px;
+
+  /*****************************************************************************
+  * Font family
+  **/
+  --font-family-base: 'Lato', sans-serif;
+  --font-family-heading: 'Open Sans', sans-serif;
+
+  /*****************************************************************************
+  * Color
+  * 
+  * Colors are defined in rgb string way, "red, green, blue"
```

Comment:
So if a user overrides those variables in their custom css, do they also always need to define the color as such a tuple?

Eg someone doing

```
:root {
  --color-primary: #333;
}
```

will work or not?

And related to my question below, the reason they are defined here as r, g, b, is that to make things like `rgba(var(--color-sidebar-link-hover), 1)` work? (if that is indeed the reason, a clarifying comment here might be helpful)

Reply:
As long as the custom CSS is added *after* the theme's CSS, it'll work. :)